src/widget/wtrackmenu.h

```diff
@@ -51,8 +51,9 @@ class WTrackMenu : public QMenu {
         SearchRelated = 1 << 13,
         UpdateReplayGainFromPregain = 1 << 14,
         SelectInLibrary = 1 << 15,
+        Reanalyze = 1 << 16,
```

Comment:
`Analyze` would be more general and implies re-analyzing.

These features are too detailed anyway, but that's a different topic.

Reply:
(Re-)Analyze?